src/main/java/piano/instruments/InstrumentButton.java

```diff
@@ -9,28 +9,27 @@ public class InstrumentButton extends JButton {
     public InstrumentButton(int instrument) {
         this.instrument = instrument;
         String text;
-        switch (instrument) {
-            case InstrumentsInterface.PIANO:
-                text = "Piano";
-                break;
-            case InstrumentsInterface.XYLOPHONE:
-                text = "Xylophone";
-                break;
-            case InstrumentsInterface.GUITAR:
-                text = "Guitar";
-                break;
-            case InstrumentsInterface.TRUMPET:
-                text = "Trumpet";
-                break;
-            case InstrumentsInterface.FLUTE:
-                text = "Flute";
-                break;
-            case InstrumentsInterface.DRUM:
-                text = "Drum";
-                break;
-            default:
-                text = "";
-                break;
+        if (instrument == Instruments.PIANO.getInstrument())
+        {
+            text = "Piano";
+        } else if (instrument == Instruments.XYLOPHONE.getInstrument())
+        {
+            text = "Xylophone";
+        } else if (instrument == Instruments.GUITAR.getInstrument())
+        {
+            text = "Guitar";
+        } else if (instrument == Instruments.TRUMPET.getInstrument())
+        {
+            text = "Trumpet";
+        } else if (instrument == Instruments.FLUTE.getInstrument())
+        {
+            text = "Flute";
+        } else if (instrument == Instruments.DRUM.getInstrument())
+        {
+            text = "Drum";
+        } else
+        {
+            text = "";
```

Comment:
This is the wrong way to use enums.

Reply:
So what is the correct way? Should I revert it to interfaces?